Dockerfile

```diff
@@ -1,15 +1,14 @@
 FROM perl:5.28.0-slim
 
-ENV PATH="/root/bystro/bin:/usr/local/go/bin:/root/go/bin/:${PATH}" \
-    PERL5LIB="/root/perl5/lib/perl5:/root/bystro/lib:${PERL5LIB}" \
-    GOPATH="/root/go"
+ENV PATH="/bystro/bin:/usr/local/go/bin:/go/bin/:${PATH}" \
+    PERL5LIB="/bystro/lib:${PERL5LIB}" \
+    GOPATH="/go"
 
-ADD ./ /root/bystro/
+COPY . /bystro
 
-WORKDIR /root/bystro
+WORKDIR /bystro
 
-RUN cpanm --local-lib=/root/perl5 local::lib && eval $(perl -I /root/perl5/lib -Mlocal::lib) \
-    && apt-get update \
+RUN apt-get update \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/app/akotlar/bystro/pullRequest?prid=4732489)